draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -243,21 +243,21 @@ The detailed algorithm is described in (#verifier_verification).
 
 This section defines data formats for SD-JWT including the Issuer-signed JWT content, Disclosures, and Key Binding JWT.
 
-## SD-JWT Payload
+## SD-JWT Input Claim Set
 
 An SD-JWT has a JWT component that MUST be signed using the Issuer's private key.
 It MUST use a JWS asymmetric digital signature algorithm. It
 MUST NOT use `none` or an identifier for a symmetric algorithm (MAC).
 
-The payload of an SD-JWT is a JSON object according to the following rules:
+The input claim set to be used as the payload of an Issuer-signed JWT component of an SD-JWT is a JSON object that:
```

Comment:
```suggestion
The input claim set to be used as the payload of the Issuer-signed JWT component of an SD-JWT is a JSON object that
```

Reply:
When ending this sentence with the conjunction "that", we need to change the enumeration below into one long sentence - I'll propose changes accordingly.

Reply:
thank you!